modules/operator-upgrade.adoc

```diff
@@ -46,6 +46,55 @@ To update {productname} from one minor version to the next, for example, 3.4 ->
 
 For `z` stream upgrades, for example, 3.4.2 -> 3.4.3, updates are released in the major-minor channel that the user initially selected during install. The procedure to perform a `z` stream upgrade depends on the `approvalStrategy` as outlined above. If the approval strategy is set to `Automatic`, the Quay Operator will upgrade automatically to the newest `z` stream. This results in automatic, rolling Quay updates to newer `z` streams with little to no downtime. Otherwise, the update must be manually approved before installation can begin.
 
+[id="upgrading-postgresql-databases"]
+=== Updating {productname} from 3.8 -> 3.9
+
+When updating {productname} 3.8 -> 3.9, the Operator automatically upgrades the existing PostgreSQL databases for Clair and {productname} from version 10 to version 13. 
+
+[IMPORTANT]
+====
+* This upgrade is irreversible. It is highly recommended that you upgrade to PostgreSQL 13. PostgreSQL 10 * will have its final release in May, 2024. Afterwards, it will be unsupported. 
```

Comment:
`PostgreSQL 10 * will` - is this correct?

Reply:
Based on https://www.postgresql.org/support/versioning/, PostgreSQL 10 already ended on  `November 10, 2022`, looks [PROJQUAY-4608](https://issues.redhat.com/browse/PROJQUAY-4608) provided wrong info